src/Chan.cpp

```diff
@@ -423,6 +430,29 @@ void CChan::ModeChange(const CString& sModes, const CNick* pOpNick) {
     }
 }
 
+void CChan::ModeChange(const CString& sModes, const CNick* pOpNick) {
+    VCString vsModes;
+    CString sModeArg = sModes.Token(0);
+    bool colon = sModeArg.TrimPrefix(":");
```

Comment:
I don't think it's job of `CChan` to handle colon. `CMessage::GetParams()` returns params without the colon already.

OTOH, probably that's fine, as long as this function is clearly marked as deprecated.

Reply:
This is handling the old behavior where a string in the form `:+i` or `+oo Nick1 :Nick2` is passed and it parses it appropriately to keep that compatibility. I will mark this implementation as deprecated as you suggest